CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 4.1.15 (Not released)
```

Comment:
@dbeltrankyl I started the blank Changelog for the next release... not sure if you had another idea for this?